ios/RNRadar.m

```diff
@@ -238,13 +219,32 @@ - (void)didLogMessage:(NSString *)message {
         rejecter = nil;
     };
 
+    NSString *accuracy = optionsDict[@"accuracy"];
+
+    if (accuracy != nil && [accuracy isKindOfClass:[NSString class]]) {
+        if ([accuracy isEqualToString:@"high"] || [accuracy isEqualToString:@"HIGH"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyHigh;
+        } else if ([accuracy isEqualToString:@"medium"] || [accuracy isEqualToString:@"MEDIUM"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyMedium;
+        } else if ([accuracy isEqualToString:@"low"] || [accuracy isEqualToString:@"LOW"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyLow;
+        }
+    }
```

Comment:
This was probably copied from elsewhere, but it's probably safest to do a case-insensitive comparison, or better yet, lowercase the `accuracy` and compare it to lowercase values.

Reply:
agree